.github/actions/update-badges/action.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Create badge'
```

Comment:
Do we still need this?

Reply:
Depends, but yes we can just put a simple job